src/patternfly/components/Switch/switch.scss

```diff
@@ -1,6 +1,7 @@
 .pf-c-switch {
   // Switch - the vars for this file don't follow our normal order so that the IE11 script can convert the vars correctly
   --pf-c-switch--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-switch--ColumnGap: var(--pf-global--spacer--md);
```

Comment:
Using this in place of the padding works fine, though if someone had customized `--pf-c-switch__label--PaddingLeft`, they're going to have double space here now.

WDYT about leaving `--pf-c-switch__label--PaddingLeft` and setting `--pf-c-switch--ColumnGap: var(--pf-c-switch__label--PaddingLeft)`?

Or maybe better, though we haven't done this before - defining this var as `--pf-c-switch--ColumnGap: var(--pf-c-switch__label--PaddingLeft, var(--pf-global--spacer--md));` and removing `--pf-c-switch__label--PaddingLeft`? 

Both of those will mean `--pf-c-switch__label--PaddingLeft` is a reserved var for the user, and not something we can use until we can clean it up in a breaking change, but it's less likely to break a user if they have customized this var.

Reply:
Love it, updated!